test/fviz_case_study/CMakeLists.txt

```diff
@@ -17,15 +17,26 @@ add_library(${PROJECT_NAME}_Test_FvizDefaultPlugin
 target_include_directories(${PROJECT_NAME}_Test_FvizDefaultPlugin
   PUBLIC "../include" ${console_bridge_INCLUDE_DIRS} ${Poco_INCLUDE_DIRS})
 target_link_libraries(${PROJECT_NAME}_Test_FvizDefaultPlugin
-  ${PROJECT_NAME} ${PROJECT_NAME}_Test_Fviz)
+  ${PROJECT_NAME} ${PROJECT_NAME}_Test_Fviz ${console_bridge_LIBRARIES} ${Poco_LIBRARIES})
 class_loader_hide_library_symbols(${PROJECT_NAME}_Test_FvizDefaultPlugin)
 
-ament_add_gtest(${PROJECT_NAME}_fviz_test fviz_test.cpp)
+set(append_library_dirs "${CMAKE_CURRENT_BINARY_DIR}")
+if(WIN32)
+  set(append_library_dirs "${append_library_dirs}/../../$<CONFIG>;${append_library_dirs}/$<CONFIG>")
```

Comment:
@wjwwood @dirk-thomas this is pretty ugly. Am I approaching this the wrong way?

Reply:
I get why you want to use `${append_library_dirs}/$<CONFIG>` but where is `${append_library_dirs}/../$<CONFIG>` supposed to be pointing to?

Maybe you can use a generator expression to point to the target directory of one of the plugin libraries instead  (like `$<TARGET_FILE_DIR:${PROJECT_NAME}_TestPlugins1>`)?